ros2_control_demo_description/rrbot_description/ros2_control/rrbot_transmissions_system_position_only.ros2_control.xacro

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="rrbot_transmissions_system_position_only" params="name prefix use_gazebo:=^|false use_fake_hardware:=^|true mock_sensor_commands:=^|false slowdown:=2.0">
+
+    <ros2_control name="${name}" type="system">
+
+      <xacro:if value="$(arg use_gazebo)">
```

Comment:
This transmission plugin does not work with gazebo, am I right? Is there a way to load the transmission plugin on top of gazebo_ros2_control? If not, I'd delete this option here in the xarco?

Reply:
You are correct, transmissions in gazebo make no sense. Same goes for the mock hardware for that matter. They were copied over from the base example and I forgot to remove them. Addressed now.

Reply:
I mean, transmissions and gazebo can make sense for example for closed-loop kinematics etc, but not in this case..